src/main/java/com/appraisal/entities/Team.java

```diff
@@ -0,0 +1,15 @@
+package com.appraisal.entities;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "teams")
+public class Team extends BaseEntity {
+    private String teamName;
```

Comment:
name is better. It's already a team's name because it is under team.
Always use context to help with naming